### PR TITLE
Add minor fixes.

### PR DESCRIPTION
**Pull request by xavierlepretre**
This PR makes minor fixes.

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [x] requested
  - [x] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [x] Ready to be merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,16 +264,16 @@ customModules:
         links:
           - title: Overview
             path: /tutorials/9-path-to-prod/1-overview.html
-          - title: Prepare the software
+          - title: Prepare the software to run
             path: /tutorials/9-path-to-prod/2-software.html
           - title: Prepare a validator and keys
             path: /tutorials/9-path-to-prod/3-keys.html
-          - title: Prepare the node start
+          - title: Prepare where the node starts
```
Comment (CitMC) [inline README.md:271]: EIther `where the node starts` or `where nodes start`. Went with the first option, if that is incorrect, please change accordingly.

**Review by CitMC — APPROVED** [review]
@xavierlepretre Please take a look at the remaining conversation, after can be merged.

File: README.md (hunk 1 above)
Comment (xavierlepretre) [reply README.md:271]: Mmh, "where nodes start" sounds good too.

**Comment by xavierlepretre** [comment]
Ok, let's merge.

**Pull request merged**